### DISCUSSION TITLE
PDF or text files upload

### DISCUSSION THREAD

**Discussion by XinyueZ** [langgenius/dify#8603] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Please add a node or component to support file uploads or upgrade `sys.files`  to other file types as early as possible. A use case like chatting with your doctor is a very common task today. You might lose your users if this feature is not available. 🥹

**Comment by IthacaDream** [comment]
Yep, I think agent-chat specifically needs this feature
